### DISCUSSION TITLE
Is there a way to increase the hover area to see the tooltip?

### DISCUSSION THREAD

**Discussion by rgowtham** [vega/altair#3597] [Q&A] [answered]
I frequently make use of tooltip to see the exact value associated with a data point (e.g., in a line graph etc). I am noticing that the hover area to get the tooltip is very small - unless you keep your mouse pointer at the exact point where the data point is - it is hard to bring up the tooltip.

Are there any configurations that I can pass to increase the hover area such that when I am in the vicinity of the data point, I can get to view the tooltip.

**Comment by kgoodrick-uu** [comment, marked as answer]
I think the easiest way to accomplish this is to assign the tooltip encoding to a second layer where the points are bigger and have an opacity of 0.

**Reply by rgowtham** [reply]
Thanks, Can you please share an example of how you will achieve this? I am relatively new to altair so an example will be of much benefit

**Reply by rgowtham** [reply]
Found an example in one of the closed issues mentioned from the link posted by @joelostblom - https://github.com/vega/altair/issues/1812#issuecomment-559101262.

**Comment by joelostblom** [comment]
I agree that this would be convenient, but it is something that would need to be implemented in Vega-Lite first, so feel free to follow / add to the discussion in https://github.com/vega/vega-lite/issues/6851. In the meantime, the workaround suggested by @kgoodrick-uu is what I occasionally use myself as well.
